pkg/gateway/cmd/server.go

```diff
@@ -257,6 +257,7 @@ func (s *server) Run(ctx context.Context, metrics metrics.Metrics, logger *zap.L
 	healthChecker := health.NewGrpcChecker(
 		health.WithTimeout(time.Second),
 		health.WithCheck("metrics", metrics.Check),
+		health.WithCheck("redis", redisV3Client.Check),
```

Comment:
By checking the Redis in the API Gateway, if it fails, it won't get the features from the DB.
But if we don't test it, DB may fail due to high requests until the container restarts.

Reply:
I removed it because we want to avoid downtime as much as possible.